src/include/daos_srv/iv.h

```diff
@@ -30,8 +30,8 @@ struct ds_iv_ns {
 	/* pool uuid */
 	uuid_t		iv_pool_uuid;
 
-	int		iv_refcount;
 	ABT_eventual	iv_done_eventual;
+	int		iv_refcount;
```

Comment:
Just to be clear, this change I proactively merged from  https://github.com/daos-stack/daos/pull/4564/  before that PR was landed to master (it has since landed but it is not worth merging master and running through CI testing in this patch).